### PR TITLE
Mark all functions in a deprecated library (like LibBytes) as deprecated (SYN-8060)

### PR DESCRIPTION
**Pull request by rakuy0**

### CODE REVIEW

File: changes/8f5c83c0f2f977bfcd07b6131aa7ad1b.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+desc: Mark all functions in a deprecated library as deprecated.
```
Comment (vEpiphyte) [inline changes/8f5c83c0f2f977bfcd07b6131aa7ad1b.yaml:2]:
```suggestion
desc: Mark all functions in a deprecated Storm library as deprecated.
```

File: synapse/tests/test_lib_storm.py [hunk 2]
```diff
@@ -3947,6 +3947,14 @@ async def test_storm_help_cmd(self):
             msgs = await core.stormlist('$mod=$lib.import(foosmod) help $mod.f')
             self.stormIsInErr('help does not currently support runtime defined functions.', msgs)
 
+            msgs = await core.stormlist('help --verbose $lib.bytes')
+            self.stormIsInPrint('Warning', msgs)
+            self.stormIsInPrint('$lib.bytes.put`` has been deprecated and will be removed in version v3.0.0', msgs)
+            self.stormIsInPrint('$lib.bytes.has`` has been deprecated and will be removed in version v3.0.0', msgs)
+            self.stormIsInPrint('$lib.bytes.size`` has been deprecated and will be removed in version v3.0.0', msgs)
+            self.stormIsInPrint('$lib.bytes.upload`` has been deprecated and will be removed in version v3.0.0', msgs)
+            self.stormIsInPrint('$lib.bytes.hashset`` has been deprecated and will be removed in version v3.0.0', msgs)
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_storm.py:3956]: Do we want to add a `mesg` to the deprecation on `LibBytes` as well? That way there is a clear redirect.

```
$ git diff
diff --git a/synapse/lib/stormtypes.py b/synapse/lib/stormtypes.py
index 6a3aac197..1bd481a56 100644
--- a/synapse/lib/stormtypes.py
+++ b/synapse/lib/stormtypes.py
@@ -2661,7 +2661,7 @@ class LibBytes(Lib):
                   'returns': {'type': 'list', 'desc': 'A tuple of the file size and sha256 value.', }}},
     )
     _storm_lib_path = ('bytes',)
-    _storm_lib_deprecation = {'eolvers': 'v3.0.0'}
+    _storm_lib_deprecation = {'eolvers': 'v3.0.0', 'mesg': 'Use the corresponding ``$lib.axon`` function.'}
 
     def getObjLocals(self):
         return {
```

**Review by vEpiphyte — CHANGES_REQUESTED** [review]
Update changelog

File: synapse/tests/test_lib_storm.py (hunk 2 above)
Comment (rakuy0) [reply synapse/tests/test_lib_storm.py:3956]: Yea, a clear redirect would be good :+1:

**Review by vEpiphyte — DISMISSED** [review]

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**